assets/thanos-querier/prometheus-rule.yaml

```diff
@@ -14,7 +14,8 @@ spec:
     rules:
     - alert: ThanosQueryHttpRequestQueryErrorRateHigh
       annotations:
-        message: Thanos Query {{$labels.job}} is failing to handle {{ $value | humanize }}% of "query" requests.
+        message: Thanos Query {{$labels.job}} is failing to handle {{ $value | humanize
```

Comment:
Sorry if this was asked before, I assume this is just because we bumped jsonnet rather than anything else?

Reply:
Yes, this is due to different versions of jsonnet used. Right now we need to manage it in two places (ci config and in repo). After this and follow-up PR in openshift/release are merged we will be managing jsonnet version in one place.

Reply:
Yes seen that PR, thanks!